src/components/elements/CommonDetailGrid.tsx

```diff
@@ -97,7 +97,6 @@ export const CommonDetailGridItem: React.FC<{
         item
         {...valueBreakPoints}
         sx={itemSx}
-        aria-labelledby={labelId}
```

Comment:
I'm proposing to remove this because the common detail grid is usually used for displaying key/value text pairs, so I think it doesn't need to aria label the values, since aria labels should be used on interactive elements. (https://www.w3.org/TR/using-aria/) This fixes an axe warning related to wcag 4.1.2. Happy to revert this though if we think the change is too far-ranging